src/custom-types.json

```diff
@@ -618,6 +618,24 @@
       "https://docs.microsoft.com/en-us/windows/win32/wic/dds-format-overview"
     ]
   },
+  "image/x-adobe-dng" : {
+    "extensions": ["dng"],
+    "sources": [
+      "https://stackoverflow.com/a/47612661/816478"
+    ]
+  },
+  "image/x-sony-arw" : {
+    "extensions": ["arw"],
+    "sources": [
+      "https://stackoverflow.com/a/47612661/816478"
```

Comment:
The sources needs to contain a [primary source](https://en.wikipedia.org/wiki/Primary_source) for the type data (https://github.com/jshttp/mime-db#contributing).